machine/machine.py

```diff
@@ -99,7 +99,10 @@ def create(self, name, driver='virtualbox', blocking=True):
         Returns:
             int: error code from the run
         """
-        cmd = ['create', '--driver', driver, name]
+        cmd = ['create', '--driver', driver]
```

Comment:
This can be written as `cmd = ['create', '--driver', driver] + xarg + [name]

Reply:
Done the changes as you suggested.